docs/pages/guides/customizing-metro.md

```diff
@@ -19,7 +26,90 @@ const config = getDefaultConfig(__dirname);
 module.exports = config;
 ```
 
-You can find a complete list of supported options in the Metro docs: [Configuring Metro](https://facebook.github.io/metro/docs/configuration). Please note that you only need to specify the options that you want to customize: the custom config will be merged with the defaults from `expo/metro-config` when using [Expo CLI][expo-cli] to start dev servers (`npx expo start`) or export static files (`npx expo export`).
+To learn more, refer to the [`metro.config.js` docs](https://facebook.github.io/metro/docs/configuration).
+
+## Assets
+
+Metro resolves files as either source code or assets. Source code is JavaScript, TypeScript, JSON, and other files that are used by your application. [Asset](/guides/assets) are images, fonts, and other files that should not be transformed by Metro. To accommodate large-scale codebases, Metro requires all extensions for both source code and assets to be explicitly defined before starting the bundler. This is done by adding the `resolver.sourceExts` and `resolver.assetExts` options to the Metro configuration. By default we include the following extensions:
+
+- [`resolver.assetExts`](https://github.com/facebook/metro/blob/main/packages/metro-config/src/defaults/defaults.js#L15)
+- [`resolver.sourceExts`](https://github.com/facebook/metro/blob/main/packages/metro-config/src/defaults/defaults.js#L15)
+
+### Adding more file extensions to `assetExts`
+
+The most common customization is including extra asset extensions to Metro. In the **metro.config.js** file, add the file extension (without a leading `.`) to `resolver.assetExts`:
+
+```js
+// metro.config.js
+const { getDefaultConfig } = require('@expo/metro-config');
```

Comment:
intentional to not use `expo/metro-config`?

Reply:
```suggestion
const { getDefaultConfig } = require('expo/metro-config');
```